src/serialization/json_object.cpp

```diff
@@ -267,7 +269,7 @@ void fromJsonValue(const rapidjson::Value& val, cryptonote::block& b)
   GET_FROM_JSON_OBJECT(val, b.major_version, major_version);
   GET_FROM_JSON_OBJECT(val, b.minor_version, minor_version);
   GET_FROM_JSON_OBJECT(val, b.timestamp, timestamp);
-  GET_FROM_JSON_OBJECT(val, b.prev_id, prev_id);
+  GET_FROM_JSON_OBJECT(val, b.prev_id, prev_hash);
```

Comment:
Things like this are just confusing things for no reason but tbh since you're the only 0mq user atm, I guess it's fine.

Reply:
I thought about this too just before the commit. I'll revert this one and the similar one below. `prev_id` isn't a terrible name.